app/models/ingredient.rb

```diff
@@ -19,24 +19,33 @@ class Ingredient < ApplicationRecord
     head
     inch
     jar
-    large
     lb
     leaf
     loaf
-    medium
     quart
     ounce
     pint
     slice
-    small
     sprig
     stalk
     stick
     tablespoon
     teaspoon
+  ].freeze
+
+  # Units that do not get pluralized
+  DESCRIPTIVE_UNITS = %w[
+    small
+    medium
+    large
     whole
   ].freeze
 
+  UNITS = [
+    STANDARD_UNITS,
+    DESCRIPTIVE_UNITS,
+  ].flatten.sort
```

Comment:
Breaks measurement units into groups based on pluralization needs, then ensures wherever `UNITS` is used that all items appear in alpha order.